### DISCUSSION TITLE
Is it possible to scroll to a particular card id?

### DISCUSSION THREAD

**Discussion by cmacdonnacha** [asmyshlyaev177/react-horizontal-scrolling-menu#228] [Q&A] [answered]
Taking [this example](https://asmyshlyaev177.github.io/react-horizontal-scrolling-menu/)

Is it possible to automatically "jump" to a particular card, given we know the `itemId`?

**Comment by asmyshlyaev177** [comment, marked as answer]
Yes, something like
```
      apiRef.current?.scrollToItem?.(
        apiRef.current?.getItemElementById('someId')
      );
```

From that example [Add item and scroll to it](https://codesandbox.io/s/basic-example-forked-3j0xm?file=/src/index.tsx)

**Reply by cmacdonnacha** [reply]
Nice thanks!
